frontend/src/core/comments/components/AddComment.css

```diff
@@ -13,13 +13,20 @@
     display: flex;
     flex-grow: 2;
     font-size: 11px;
-    height: auto;
+    min-height: 26px;
+    max-height: 144px;
     line-height: 24px;
     overflow: auto;
-    padding: 6px;
+    padding: 5px;
     resize: none;
 }
 
+.public-DraftEditorPlaceholder-inner {
+  position: absolute;
```

Comment:
That causes the placeholder to stay at the same position as I scroll the History panel:

<img width="299" alt="Screenshot 2020-04-07 at 17 35 37" src="https://user-images.githubusercontent.com/626716/78689096-3e41a980-78f6-11ea-8140-4cb703ed0b21.png">

Reply:
I wasn't able to find an immediate solution here. I will continue to work on this.

Reply:
I believe this is now fixed.

Reply:
Thanks. Setting width to a fixed value will likely cause problems with localizability. Can we make it relative, i.e. to occupy the entire width available?

I wonder if [Draft.css](https://draftjs.org/docs/advanced-topics-issues-and-pitfalls/#missing-draftcss) can help with styling issues? I just came across it in the docs, and I didn't dig deeper, so I'm likely to be off. :)